his/core/mssna-printsessionappc-class2.md

```diff
@@ -86,12 +86,12 @@ class MsSna_PrintSessionAppc : MsSna_PrintSession
  **AS400Device**  
  Data Type: **String** Qualifiers: **MAXLEN(10), TOUPPERCASE** Access Type: Read/Write  
   
- The name for the AS/400 printer device. **AS400Device** should be a descriptive name that distinguishes different printers on the network.  
+ The name for the IBM System i printer device. **AS400Device** should be a descriptive name that distinguishes different printers on the network.  
```

Comment:
Is the property still named **AS400Device"?

Reply:
Yes, let's leave the property as is. I spoke with Tammi asking her to start changing the HIS product code to align with the docs changes.